jenkins/pipeline.groovy

```diff
@@ -105,7 +105,7 @@ pipeline {
         compressBuildLog()
         skipDefaultCheckout()
         skipStagesAfterUnstable()
-        timeout(time: 10, unit: 'HOURS')
+        timeout(time: 15, unit: 'HOURS')
```

Comment:
This is global right? Possible to limit to Valgrind jobs only?

Reply:
yes, should be possible to use variable with default value

Reply:
On the other hand, looking at sanitizer run results, please set 15 h there too